nifpga/bitfile.py

```diff
@@ -171,6 +181,61 @@ def __str__(self):
                 "\tOffset: %d\n" % self._offset)
 
 
+class Fxp_Register(Register):
+    """
+    A fixed point control or indicator from the front panel of the top level
+    FPGA VI.
+    """
+    def __init__(self, reg_xml, register):
+
+        self._copy_values_from_register(register)
+        datatype = reg_xml.find("Datatype")
+        fxp_xml = datatype.find("FXP")
+        self._signed = True if fxp_xml.find("Signed").text.lower() == 'true' else False
+        self._overflow = True if fxp_xml.find("IncludeOverflowStatus").text.lower() == 'true' else False
+        self._word_length = int(fxp_xml.find("WordLength").text)
```

Comment:
What happens if the XML is malformed, such as a missing WordLength tag, or a value that cannot be reasonably interpreted as an integer?

Reply:
If the bitfile is bad or somehow got corrupt (missing tag or can not be reasonably interpreted), the functions will raise an exception. As a user this basically means they cannot open a session.